prow/gerrit/adapter/adapter.go

```diff
@@ -692,6 +697,28 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 	return nil
 }
 
+// lastRevisionWithCodeChange tries to find a revision with code change since last update.
+// There is no point repeating tests if no code change happened since last update.
+func lastRevisionWithCodeChange(change client.ChangeInfo, lastUpdate time.Time) (client.RevisionInfo, error) {
+	var revisionsWithCodeChange []client.RevisionInfo
+	for _, revision := range change.Revisions {
+		if revision.Kind != gerrit.NoCodeChange {
```

Comment:
Nit: I prefer positive boolean checks where possible. This currently reads "if not no code change ..." which is awkward. Easier to do "if revision.Kind == gerrit.NoCodeChange { continue }" and then move the append line outside this block.

Reply:
Done.